### PR TITLE
Delete and cloning migration.

### PR DESCRIPTION
**Pull request by joshirohit100**
cc @amitgoyal  @pranitjha  for Deleting and clone migration.

screen shot 2018-12-05 at 7 42 00 pm
screen shot 2018-12-05 at 7 42 20 pm
screen shot 2018-12-05 at 7 42 35 pm
screen shot 2018-12-05 at 7 42 44 pm
screen shot 2018-12-05 at 7 42 55 pm

### CODE REVIEW

File: src/Form/ConfirmMigrationCloneForm.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    }
+
+    return $migration_id;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() : string {
+    return "confirm_delete_form";
```
Comment (amitgoyal) [inline src/Form/ConfirmMigrationCloneForm.php]: This should be clone rather delete.

**Comment by amitgoyal** [comment]
Change the migration title after it gets cloned in both listing/msg.

**Comment by amitgoyal** [comment]
See if we should add Edit migration link in the List migration.

File: src/Form/ConfirmMigrationCloneForm.php (hunk 1 above)
Comment (joshirohit100) [reply src/Form/ConfirmMigrationCloneForm.php]: PR updated! Thanks!

**Comment by joshirohit100** [comment]
PR is updated! Now also has edit migration link

**Comment by joshirohit100** [comment]
cc @amitgoyal

**Comment by amitgoyal** [comment]
LGTM

**Pull request merged**